views/login/eligibility-children.pug

```diff
@@ -0,0 +1,18 @@
+extends ../base
+
+block variables
+  -var title = __('Children under 18')
+  -var children = hasData(data, 'login.children') ? data.login.children : ''
+
+block content
+
+  +stepper('3', '7')
+
+  h1 #{title}
+  
+  form.cra-form(method='post')
+    +radiosYesNo('children', 'Do you have a child or children under age 18?')
```

Comment:
Don't forget to pass in the values for the rest of 'em. (here it would be `children `.)